instances/widgets.treasury-factory.near/widget/components/TokenAmount.jsx

```diff
@@ -7,12 +7,20 @@ const amountWithDecimals = props.amountWithDecimals ?? 0;
 const amountWithoutDecimals = props.amountWithoutDecimals;
 
 const isNEAR = address === "" || address.toLowerCase() === "near";
+const isWrapNear = address === "wrap.near";
 
 let ftMetadata = {
   symbol: "NEAR",
   decimals: 24,
 };
-if (!isNEAR) {
+if (isWrapNear) {
+  ftMetadata = {
```

Comment:
Maybe just put a comment in the code here that the reason for hardcoding the metadata for wNEAR is that the icon is not available when calling the `ft_metadata` view method on wrap.near